### ISSUE TITLE
[A11y] - Hidden or empty element receives focus

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1395]
### Current Behaviour 
`Keyboard focus or touch screen swiping falls on an element that is hidden or empty. `<br/><hr/>
### Expected Behaviour 
`RULE :  The reading and navigation order MUST be logical and intuitive. All focusable elements MUST have a visual focus indicator when in focus. HOW TO FIX:  Fix this issue by using ONE of the following techniques: 1.If the element should not be visible or focusable, remove the element from the DOM or hide it from all users with CSS such as display:none. 2.If the element is meant to be visible but not interactive, make it visible and remove the focus state. 3.If the element is meant to be visible and interactive, then ensure that it is both focusable and visible to all users. `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [x] Serious
- [ ] Critical
- [ ] Moderate
- [ ] Minor

### Issue Details

<details><summary>View Issue 1513116 </summary> 
 <p>

_**Auditor ID**_ : [1513116](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1513116&searchOn=id) 

_**description**_ :  Example copied to clipboard dismiss button receives focus but is not visible to the keyboard-only/ visible users.

Element location: On the "Accordion" page

Steps to reproduce: 
- Without dismissing the "example copied to clipboard" status message, trigger another Copy button. 
-The same message appears below the existing message. So when navigating using a tab, the dismiss button receives focus but is not visible to the keyboard-only/ visible users.

Note: It is applicable to all the pages 

_**Page Name**_ : Project Wide 

_**source**_ : `<button type="button" class="Toast__dismiss" aria-label="Dismiss"><span aria-hidden="true" class="Icon Icon--close">` 

_**Screenshots**_ : [Keyboard focus order](https://axeauditor.dequecloud.com/api/v1/file/df9d6d7e-5d0d-11ee-816d-036a4d3bb847) 

</p>
</details>

**Comment by scurker** [comment]
This one, we probably want to limit one toast at a time for copy buttons. There's a larger issue at hand where we need to limit the toasts that can be shown or stack toasts visually that will need to be addressed elsewhere.
